test/integration/asymmetric-encrypt-decrypt.int.c

```diff
@@ -100,13 +100,12 @@ test_invoke (TSS2_SYS_CONTEXT *sapi_context)
 
     // First clear attributes bit field.
     *(UINT32 *)&(in_public.publicArea.objectAttributes) = 0;
-    in_public.publicArea.objectAttributes.restricted = 0;
-    in_public.publicArea.objectAttributes.userWithAuth = 1;
-    in_public.publicArea.objectAttributes.decrypt = 1;
-    in_public.publicArea.objectAttributes.sign = 1;
```

Comment:
You missed a `in_public.publicArea.objectAttributes |= TPMA_OBJECT_SIGN` here.

Reply:
Awesome catch... Thanks, fixed...